packages/docs/src/components/header/components/algolia-search/AlgoliaSearch.vue

```diff
@@ -27,30 +9,22 @@ import { Options, Vue } from 'vue-class-component'
   name: 'Search',
 })
 export default class Search extends Vue {
-  value = ''
-  color = 'gray'
-
-  onFocusHandler (colorValue: string, displayValue: string): void {
-    this.color = colorValue
-    const el: HTMLElement | null = document.querySelector('.ds-dropdown-menu')
-    if (el) { el.style.display = displayValue }
-  }
-
   initialize () {
     Promise.all([
-      // @ts-ignore
-      import(/* webpackChunkName: "docsearch" */ 'docsearch.js/dist/cdn/docsearch.min.js'),
-      // @ts-ignore
-      import(/* webpackChunkName: "docsearch" */ 'docsearch.js/dist/cdn/docsearch.min.css'),
-    ]).then(([docsearch]) => {
-      docsearch = docsearch.default
-      docsearch(Object.assign({
-        apiKey: 'be3528055c92da2ea5133b93ed548c6d',
+      import(/* webpackChunkName: "docsearch" */ '@docsearch/js'),
```

Comment:
```
const {docsearch: default} = await import(/* webpackChunkName: "docsearch" */ '@docsearch/js')
```

would do the same? Looks cleaner to me.

Reply:
I'm really not sure that we need here dynamic import, so jsut replace it by ordinary static import

Reply:
Dynamic import was to speed up initial page load (I think). Basically algolia is included in separate bundle and is sideloaded.